modules/core/src/main/java/org/apache/ignite/internal/processors/query/schema/SchemaOperationWorker.java

```diff
@@ -177,8 +178,17 @@ public boolean cacheRegistered() {
      * Cancel operation.
      */
     @Override public void cancel() {
-        if (cancelToken.cancel())
+        if (cancelToken.cancel()) {
+            try {
+                fut.get();
+            }
+            catch (IgniteCheckedException e) {
+                if (log.isDebugEnabled())
```

Comment:
I think when you use isDebugEnable() need printing via debug(...).
But here warning will be better.